### ISSUE TITLE
Temporary lock up when moving/resizing window

### ISSUE THREAD

**Issue by chaseadam** [vibou/vibou.gTile#50]
I experience this quite often with a few windows on Fedora 24. Recently I have seen this with Firefox and ScudCloud. The lock up can last anywhere from a few seconds to 20 seconds. The mouse is still usable, but I am unable to interact with gnome or any windows (keyboard or otherwise). The gTile screen is showing during this time with the desired tiles selected.

I do have a multi-monitor setup, but I was able to experience this with a single monitor setup as well (so not #47). This may be related to #20 but I am not getting a crash.

**Comment by pycy** [comment]
me too.  I have exactly the same problem but lockup can be indefinite.

**Comment by stofstik** [comment]
Same here. Just a guess, but it looks like some sort of memory leak. The first few times gTile works fine, then eventually it freezes.

**Comment by pycy** [comment]
while we're all here then, anecdotally, it seems to freeze with a higher probability when i am trying to resize a gnome terminal window.  (take with grain of salt, could be nothing...)

**Comment by glogiotatidis** [comment]
> Same here. Just a guess, but it looks like some sort of memory leak. The first few times gTile works fine, then eventually it freezes.

Agree. Restarting gnome shell (with alt-f2 r) and gTile fixes the issue temporarily

**Comment by mehaase** [comment]
I have this problem predominantly with Chrome.
